lib/media/media_source_engine.js

```diff
@@ -164,8 +164,9 @@ shaka.media.MediaSourceEngine = class {
     /** @private {?number} */
     this.lastDuration_ = null;
 
-    /** @private {?shaka.util.TsParser} */
-    this.tsParser_ = null;
+    /** @private {!Object.<shaka.util.ManifestParserUtils.ContentType,
+                           !shaka.util.TsParser>} */
+    this.tsParser_ = {};
```

Comment:
Should be renamed `this.tsParsers_`, now that it contains multiple TS Parsers.

Reply:
Done!